frontend/pages/admin/IntegrationsPage/cards/Mdm/_styles.scss

```diff
@@ -0,0 +1,35 @@
+.mdm-integrations {
+  display: flex;
+  flex-direction: column;
+  width: 65%;
+  gap: 80px;
+
+  &__section {
+    @include clearfix;
```

Comment:
is clearfix this needed? its usually for floated items isnt it? but I dont see any use of floats here.